gapis/resolve/memory.go

```diff
@@ -65,7 +65,10 @@ func Memory(ctx context.Context, p *path.Memory) (*service.Memory, error) {
 	list.Atoms[atomIdx].Mutate(ctx, s, nil /* no builder, just mutate */)
 
 	slice := pool.Slice(r)
-	observed := slice.ValidRanges()
+
+	if !p.ExcludeOtherObservations {
+		observed = slice.ValidRanges()
```

Comment:
It seemed large... a few thousands. It gets requested every time the user selects an atom/node, so I wanted it to be fast. I am fine to drop this change though.